paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlTypeUtils.java

```diff
@@ -262,6 +263,9 @@ public static DataType toDataType(
             case TEXT:
             case MEDIUMTEXT:
             case LONGTEXT:
+                return length == null || length == 0 || typeMapping.containsMode(LONGTEXT_TO_BYTES)
+                        ? DataTypes.VARCHAR(length)
+                        : DataTypes.STRING();
```

Comment:
DataTypes.VARBINARY?

Reply:
Sorry, this is WIP and the code has not been submitted.

Reply:
There's an issue with my VPN, so I won't be able to submit the code today.😭